ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-forms.view.js

```diff
@@ -43,9 +43,7 @@ module.exports = SearchFormViews.extend({
       return
     }
 
-    lightboxInstance.model.updateTitle(
-      this.model.get('type') === 'new-result' ? '' : this.model.get('name')
-    )
+    lightboxInstance.model.updateTitle(this.model.get('title'))
```

Comment:
we no longer need the type check here?

Reply:
The default title for a new result form, as defined in `search-form.js`, is `''`, so this check isn't necessary